### ISSUE TITLE
Sonoff zb bridge and zb sensors

### ISSUE THREAD

**Issue by jbega** [AlexxIT/SonoffLAN#676]
Hello!
I have noticed that all my sonoff devices work in local and cloud mode except the zigbee bridge (sonoff too) and the temperature sensors it manages (sonoff too). This sensor only work on cloud mode
Do you have to do any extra config for these devices? I see the configs documentation for RF bridge (but this is not my model)
Thanks

**Comment by yvesdemoerloose** [comment]
+1 same problem. Stopped working since latest HA version.  All the wifi devices are online and work as before, only zigbee looks affected. 


Logs showing: 
Logger: custom_components.sonoff.sonoff_cloud
Source: custom_components/sonoff/sonoff_cloud.py:256
Integration: Sonoff (documentation, issues)
First occurred: January 13, 2022, 23:31:12 (1 occurrences)
Last logged: January 13, 2022, 23:31:12

Cloud WS Connection error: Cannot connect to host 52.59.155.15:8080 ssl:False [Connect call failed ('52.59.155.15', 8080)


Kind regards
Y

**Comment by jbega** [comment]
Sorry, yvesdemoerloos I did not express myself correctly, English is not my language. My zb sensors work perfectly in HA in cloud mode but not in local mode. I realized this last night (around the same time as your error) because the sonoff server in my area was out and my lights and switches were working with HA but the zigbee devices were not. Looking at the attributes I saw that it only put "cloud mode"
